amadeus-integration-test/TravelRestrictionsTest.cs

```diff
@@ -13,7 +13,10 @@ public class TravelRestrictionsTest
         public void Get_Restrictions()
         {
             var amadeus = GetAmadeusBuild();
-            DiseaseAreaReport response = amadeus.dutyOfCare.diseases.covid19AreaReport.get(Params.with("countryCode", "US"));
+            DiseaseAreaReport response = amadeus.dutyOfCare.diseases.covid19AreaReport.get(Params
```

Comment:
The Travel Restrictions API has been recently completely decommissioned, so there is no need to update this anymore.